multicluster/test/e2e/service_test.go

```diff
@@ -16,17 +16,22 @@ package e2e
 
 import (
 	"fmt"
+	"math/rand"
 	"strconv"
+	"strings"
 	"testing"
 	"time"
 
+	log "github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 
 	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
 	antreae2e "antrea.io/antrea/test/e2e"
 	e2euttils "antrea.io/antrea/test/e2e/utils"
 )
 
+var nodeNames map[string]string
```

Comment:
This should be stored in `*MCTestData` and probably name it `clusterGateways`

Reply:
Done